### ISSUE TITLE
Uncaught TypeError: Class constructor FuzzyFinderView cannot be invoked without 'new'

### ISSUE THREAD

**Issue by thancock20** [viddo/recent-files-fuzzy-finder#20]
[Enter steps to reproduce:]

1. Be on Atom 1.16 beta
2. Toggle recent-files-fuzzy-finder
This is probably related to the fuzzy-finder [replacing atom-space-pen-views with select-list](https://github.com/atom/fuzzy-finder/pull/273)

**Atom**: 1.16.0-beta0 x64
**Electron**: 1.3.13
**OS**: Linux Mint
**Thrown From**: [recent-files-fuzzy-finder](https://github.com/viddo/recent-files-fuzzy-finder) package 0.3.0

### Stack Trace

Uncaught TypeError: Class constructor FuzzyFinderView cannot be invoked without 'new'

```
At /home/todd/.atom/packages/recent-files-fuzzy-finder/lib/recent-files-view.coffee:6

TypeError: Class constructor FuzzyFinderView cannot be invoked without 'new'
    at /packages/recent-files-fuzzy-finder/lib/recent-files-view.coffee:6:7)
    at Object.createRecentFilesView (/packages/recent-files-fuzzy-finder/lib/main.coffee:38:30)
    at /packages/recent-files-fuzzy-finder/lib/main.coffee:18:10)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -7:03.5.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.editor-background.theme-oceanic-next.theme-glacier-dark-ui)
     -7:00.7.0 tree-view:toggle (atom-workspace.workspace.scrollbars-visible-always.editor-background.theme-oceanic-next.theme-glacier-dark-ui)
     -6:31.8.0 vim-mode-plus:move-down (input.hidden-input)
     -6:30.8.0 vim-mode-plus:move-up (input.hidden-input)
     -6:29.3.0 vim-mode-plus:activate-characterwise-visual-mode (input.hidden-input)
  2x -6:28.6.0 vim-mode-plus:move-down (input.hidden-input)
  9x -6:25.6.0 vim-mode-plus:move-right (input.hidden-input)
     -6:18.7.0 vim-mode-plus:activate-characterwise-visual-mode (input.hidden-input)
     -6:15.7.0 vim-mode-plus:activate-linewise-visual-mode (input.hidden-input)
  4x -6:15.3.0 vim-mode-plus:move-up (input.hidden-input)
     -6:13.1.0 vim-mode-plus:activate-linewise-visual-mode (input.hidden-input)
     -5:40.8.0 dblclick (div.line)
     -0:03.1.0 recent-files-fuzzy-finder:toggle-finder (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.5 
api-docs 0.0.6 
atom-beautify 0.29.17 
atom-clock 0.1.6 
atom-jade 0.3.0 
atom-live-server 2.1.2 
atom-ternjs 0.18.2 
atom-unsaved-changes 0.4.0 
atomic-chrome 0.3.0 
autoclose-html 0.23.0 
autocomplete-modules 1.6.7 
bang 2.1.0 
bracket-padder 0.4.1 
break 2.0.2 
busy-signal 1.3.0 
clip-history 0.3.0 
clipboard-swap 0.3.1 
cursor-history 0.9.1 
disable-keybindings 0.6.0 
docblockr 0.9.1 
double-tag 0.4.0 
editor-background 1.2.38 
emmet 2.4.3 
fancy-bracket-matcher 1.0.1 
file-icons 2.0.17 
geiger 2.1.0 
git-diff-details 1.4.0 
git-history 3.2.0 
git-time-machine 1.5.6 
glacier-dark-ui 0.5.0 
hyperclick 0.0.40 
intentions 1.1.2 
js-hyperclick 1.10.1 
jumpy 3.1.3 
language-babel 2.56.2 
language-diff 0.7.0 
language-markdown 0.19.1 
language-tt2 0.1.0 
linter 1.11.23 
linter-alex 3.2.0 
linter-csslint 1.3.4 
linter-eslint 8.1.3 
linter-just-say-no 0.7.0 
linter-ruby 1.2.5 
linter-stylint 2.2.6 
linter-write-good 0.8.0 
livestyle-atom 0.2.9 
local-web-server 0.1.0 
markdown-lists 0.0.2 
markdown-preview-enhanced 0.10.8 
markdown-table-formatter 2.9.0 
merge-conflicts 1.4.4 
narrow 0.37.1 
nerd-treeview 0.4.3 
oceanic-next 1.0.0 
octocat-syntax 1.3.1 
open-this 0.4.0 
outline-selection 0.3.1 
pigments 0.39.0 
quick-highlight 0.8.1 
recent-files-fuzzy-finder 0.3.0 
relative-numbers 0.7.1 
ruby-slim 0.2.0 
simple-align 0.2.0 
smart-quotes-plus 2.1.0 
smart-tab-name 1.0.5 
smart-tags 0.1.5 
Stylus 3.1.0 
sync-settings 0.8.1 
tab-numbers 0.6.0 
todo-show 1.11.0 
tree-view-full-width 0.1.0 
tree-view-git-status 1.3.1 
turbo-javascript 1.4.0 
vim-mode-plus 0.83.0 
vim-mode-plus-duplicate 0.2.0 
vim-mode-plus-exchange 0.2.1 
vim-mode-plus-keymaps-for-surround 0.2.1 
vim-mode-plus-move-selected-text 0.8.1 
vim-mode-zz 0.1.1 
wordcount 2.10.4 
```

**Comment by viddo** [comment]
Thanks for the report! Yes, most likely related to the referenced changeset.

I hope to to solve it during the weekend sometime unless someone beats me to it.

**Comment by thancock20** [comment]
Sounds great. This is one of those packages that I don't think about how often I use it, until it started throwing errors.

**Comment by viddo** [comment]
Was fixed in #21, released in [v0.3.1](https://github.com/viddo/recent-files-fuzzy-finder/releases/tag/v0.3.1)
